de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/NegotiationPacketListener.java

```diff
@@ -265,7 +265,7 @@ private void sessionNegotiationRequest(final JID sender,
     private void projectNegotiationCanceled(final JID sender,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this method into the anonymous class declared at line 110. ([squid:S3398](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3398))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJqzvkEeD8QRkTSt4&open=AWcmJqzvkEeD8QRkTSt4)

[ISSUE_KEY]: AWcmJqzvkEeD8QRkTSt4